docker/cygnus-common/README.md

```diff
@@ -1,164 +1,3 @@
-{"success":"true","result" : { SubscriptionID = 573ac3b6aba73680b1905f5c}
-```
-
-You will be able to see something like the following in the cygnus-common terminal:
-
-```
-time=2016-05-17T07:09:40.956UTC | lvl=INFO | corr= | trans= | srv= | subsrv= | function=handle | comp=cygnus-common | msg=com.telefonica.iot.cygnus.management.ManagementInterface[131] : Management interface request. Method: POST, URI: /v1/subscriptions
-time=2016-05-17T07:09:41.343UTC | lvl=INFO | corr=dfca71a9-41f1-4f98-8fa8-4f15612a72db | trans=dfca71a9-41f1-4f98-8fa8-4f15612a72db | srv= | subsrv= | function=<init> | comp=cygnus-common | msg=com.telefonica.iot.cygnus.backends.http.HttpClientFactory[79] : Setting max total connections (500)
-time=2016-05-17T07:09:41.343UTC | lvl=INFO | corr=dfca71a9-41f1-4f98-8fa8-4f15612a72db | trans=dfca71a9-41f1-4f98-8fa8-4f15612a72db | srv= | subsrv= | function=<init> | comp=cygnus-common | msg=com.telefonica.iot.cygnus.backends.http.HttpClientFactory[80] : Setting default max connections per route (100)
-```
-
-You can stop the container as:
-
-```
-$ docker stop c88bc1b66cdc
-c88bc1b66cdc
-$ docker ps
-CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
-```
-
-[Top](#top)
-
-###<a name="section3.2"></a>Using a specific configuration
-As seen above, the default configuration distributed with the image is tied to certain values that may not be suitable for you tests. Specifically:
-
-* The logging level is `INFO`.
-* The logging appender is `console`.
-
-You may need to alter the above values with values of your own.
-
-[Top](#top)
-
-####<a name="section3.2.1"></a>Editing the docker files
-The easiest way is by editing both the `Dockerfile` and/or `agent.conf` file under `docker/cygnus-common` and building the cygnus-common image from the scratch.
-
-This gives you total control on the docker image.
-
-[Top](#top)
-
-####<a name="section3.2.2"></a>Environment variables
-Those parameters associated to an environment variable can be easily overwritten in the command line using the `-e` option. For instance, if you want to change the log4j logging level, simply run:
-
-    $ docker run -e LOG_LEVEL='DEBUG' cygnus-common
-
-[Top](#top)
-
-####<a name="section3.2.3"></a>Using volumes
-Another possibility is to start a container with a volume (`-v` option) and map the entire configuration file within the container with a local version of the file:
-
-    $ docker run -v /absolute/path/to/local/agent.conf:/opt/apache-flume/conf/agent.conf cygnus-common-1
-
-Of course, you can combine volumes and environment variables overwriting:
-
-    $ docker run -v /absolute/path/to/local/agent.conf:/opt/apache-flume/conf/agent.conf -e LOG_LEVEL='DEBUG' cygnus-common
-
-[Top](#top)
+Please, refer to [root docker documentation](../../../doc/cygnus-common/installation_and_administration_guide/install_with_docker.md) if you want to use a docker image for cygnus-common.
```

Comment:
This about root... what does it means in this context? I think something like "Please, refer to the documentation if you want..."